task/net/build.rs

```diff
@@ -49,10 +49,14 @@ fn generate_net_config(
         writeln!(
             out,
             "{}",
-            generate_socket_state(name, socket, config.vlan.map(|v| v.count))?
+            generate_socket_state(
+                name,
+                socket,
+                config.vlan.map(|v| v.count).unwrap_or(1)
```

Comment:
How do you feel about `config.vlan.map_or(1, |v| v.count)`?

John introduced me to it, and it's a bit shorter, but doesn't read quite as naturally (maybe just because I haven't seen it as often).

Reply:
`map_or` has its arguments in the wrong order and I find it jarring.